<launch>

    <!-- visaulize the underwater urdf robot model-->
    <arg name="model" default="$(find underwater_sim)/urdf/underwater.urdf"/>

    <param name="robot_description" command="cat $(arg model)" />
        
    <node name="robot_broadcast" pkg="underwater_sim" type="robot_broadcast.py"/>
    <!-- send fake joint values -->
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <param name="use_gui" value="false"/> 
    </node>

    <!-- Combine joint values -->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>


</launch>
